tests/fixtures/admon.md

```diff
@@ -267,3 +267,32 @@ Does not render
 <p>!!!
 content</p>
 .
+
+
+
+MKdocs Closed Collapsible Sections
+.
+??? note
+    content
+.
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>content</p>
+</div>
+.
```

Comment:
The collapsible syntax for MkDocs is only rendered when the plugin is specified, but the "???" is part of the core syntax: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks

Supporting "???" in `mdit-py-plugins` means that my formatter can autoformat it here: https://github.com/KyleKing/mdformat-admon/blob/2b55fb462fac5e4e6b1abfad1f3d27c75bd1209c/tests/fixtures.md?plain=1#L239-L255

I figure the other formats for admonitions and callouts are out of scope for `mdit` to support: https://github.com/KyleKing/mdformat-admon/pull/4#discussion_r1049582215

Reply:
FYI: This is tested and working in my `mdformat-admon` plugin: https://github.com/KyleKing/mdformat-admon/blob/e64c15ef966e762915714cb3025a06c7854ed36a/mdformat_admon/index_pr58.py